projects/storefrontapp-e2e-cypress/cypress/support/general.commands.ts

```diff
@@ -44,3 +61,127 @@ Cypress.Commands.add(
       .click();
   }
 );
+
+export interface LoginUserData {
+  firstName?: string;
+  lastName?: string;
+  alias: string;
+  password?: string;
+  titleCode?: string;
+  username?: string;
+}
+
+export interface LoginOptions {
+  alwaysNew?: boolean;
+}
+
+Cypress.Commands.add(
+  'requireLoggedIn',
```

Comment:
As discussed offline, we rather don't need sharing users between test suites. So we can rename this function to `registerNewUser` and provide less details to it (if any).

Reply:
After offline discussion, it should stay as an option for future performance improvements - to wisely share users between test suitest (when test suites don't interfere with each other)

Reply:
According to offline discussion I kept current shared user behavior (with little improvement) and did some changes to unique user flow.